The Debian building host doesn't have full network connectivity
So we don't run unit test that needs such.
--- ls-registration-daemon.orig/t/discover_primary_ls.t	2019-04-12 12:03:48.520391000 +0000
+++ ls-registration-daemon/t/discover_primary_ls.t	2019-05-03 13:15:06.805999000 +0000
@@ -2,8 +2,7 @@
 
 use strict;
 use warnings;
-use FindBin qw($Bin);
-use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
@@ -17,11 +16,11 @@
 ok( $ls_instance = discover_primary_lookup_service(), "discover_primary_lookup_service completed");
 
 #Make sure result is defined
-ok(defined $ls_instance, "LS instance defined") 
-    or diag("No LS discovered. You may want to check your network connection.");
+#ok(defined $ls_instance, "LS instance defined") 
+#    or diag("No LS discovered. You may want to check your network connection.");
 
 #Make sure result is non-empty
-ok($ls_instance ne '', "LS instance is non-empty");
+#ok($ls_instance ne '', "LS instance is non-empty");
 
 #Make sure it starts with http:// or https://
-like($ls_instance, qr/https?\:\/\/.+/, "LS instance looks like a http/https URL");
+#like($ls_instance, qr/https?\:\/\/.+/, "LS instance looks like a http/https URL");
